### PR TITLE
Replaces arrival sound 

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Title
~~This was entirely made by Kate Bishop, all credit (and 1 tf2 key) goes to him.~~ lol no
The sound is from tik tok video of [Nathan_does_voices](https://www.instagram.com/nathan_does_voices/?hl=en), ~~I will be reaching out to him to ask if he is fine with us using it.~~ Nathan has responded and gave me his permission to use the voice in game.

alternative to and closes #2802
## Why It's Good For The Game

We have our own arrival sound
pog

## Changelog
:cl: Nathan_does_voices
add: New arrival sound made by Nathan_does_voices (be sure to look him up on instagram or tik tok!)
/:cl:

### CODE REVIEW

**Comment by katebishop49** [comment]
pog

**Comment by TeomanTheGreat** [comment]
What does it sound like

**Comment by Archanial** [comment]
> What does it sound like

There is person speaking, and some stutter and it ends.

**Comment by nikk735** [comment]
The original sound is actually from https://discordapp.com/channels/427337870844362753/427337870844362755/770471106528280577

File: code/modules/mob/dead/new_player/new_player.dm [hunk 1]
```diff
@@ -310,7 +310,7 @@
 		if(!arrivals_docked)
 			var/obj/screen/splash/Spl = new(character.client, TRUE)
 			Spl.Fade(TRUE)
-			character.playsound_local(get_turf(character), 'sound/voice/ApproachingTG.ogg', 25)
+			character.playsound_local(get_turf(character), 'sound/voice/welcomeBee.ogg', 50)
```
Comment (ike709) [inline code/modules/mob/dead/new_player/new_player.dm:313]: Have you checked the volume level in game? The sound file already sounds pretty quiet.

Comment (Archanial) [reply code/modules/mob/dead/new_player/new_player.dm:313]: I did, sounds good for me, but you know, volume is fucky wucky...

**Comment by heepox** [comment]
so its been 15 days... this getting merged or test merged anytime this month?

**Comment by Blueturbo47** [comment]
> so its been 15 days... this getting merged or test merged anytime this month?

21 days, going to a month. Hope it gets merged though.

**Comment by Crossedfall** [comment]
Do we have permission from Nathan_Does_Voices to use this clip? It's my understanding that they weren't paid for this and did it for a laugh, but are they aware of the intended use?

**Comment by Crossedfall** [comment]
An additional blocker would be that I don't see any proper author credits given to Nathan, who provided the actual voice line.

**Review by Crossedfall — CHANGES_REQUESTED** [review]
In addition to the prior blocker, I don't see any proper author credits given to Nathan; who lent their voice to make the clip possible. (Not Kate)

Marking as DNM until these two blockers are clarified/resolved.

**Review by Crossedfall — APPROVED** [review]
Confirmed that they gave permission for our use and are being properly attributed for their contribution. All good on my end.

**Comment by qwertyquerty** [comment]
Welcome to the beestation

Only play on bee sage

**Pull request merged**